### PR TITLE
remove duplicate of ditto

### PR DESCRIPTION
**Pull request by OnyxAbove**
The program Ditto had a duplicate entry.

I removed:
```
"WPFInstallditto": {
		"category": "Utilities",
		"choco": "ditto",
		"content": "Ditto",
		"description": "Ditto is an extension to the standard windows clipboard.",
		"link": "https://ditto-cp.sourceforge.io/",
		"winget": "Ditto.Ditto"
},
```
In favor of:
```
"WPFInstallditto": {
		"category": "Utilities",
		"choco": "ditto",
		"content": "Ditto (Clipboard Manager)",
		"description": "Ditto is an extension to the Windows Clipboard. You copy something to the Clipboard and Ditto takes what you copied and stores it in a database to retrieve at a later time.",
		"link": "https://github.com/sabrogden/Ditto",
		"winget": "Ditto.Ditto"
},
```
Because I think that the title "Ditto (Clipboard Manager)" is more helpful than just "Ditto" and that it is more useful to be linked to the GitHub rather than the SourceForge.

### CODE REVIEW

**Comment by og-mrk** [comment]
Hello @OnyxAbove 
Thanks for opening this PR, but I do want to inform you that there's a PR which does this change, PR  https://github.com/ChrisTitusTech/winutil/pull/2169 PR by @Owen-3456

With that being said.. if for whatever reason that PR by Owen is rejected, then this PR would probably get merged (personally, I can't see why _this_ PR might get rejected), so it's better to keep it open for Chris to review 👍

File: config/applications.json [hunk 1]
```diff
@@ -407,13 +407,6 @@
...
 		"winget": "Discord.Discord"
 	},
-	"WPFInstallditto": {
-		"category": "Utilities",
-		"choco": "ditto",
-		"content": "Ditto",
-		"description": "Ditto is an extension to the standard windows clipboard.",
-		"link": "https://ditto-cp.sourceforge.io/",
-		"winget": "Ditto.Ditto"
 	},
```
Comment (og-mrk) [inline config/applications.json:410]:
```suggestion
```
Forgot to remove the closing curly bracket.

**Comment by OnyxAbove** [comment]
> Hello @OnyxAbove Thanks for opening this PR, but I do want to inform you that there's a PR which does this change, PR #2169 PR by @Owen-3456
> 
> With that being said.. if for whatever reason that PR by Owen is rejected, then this PR would probably get merged (personally, I can't see why _this_ PR might get rejected), so it's better to keep it open for Chris to review 👍

Hello, sorry for that @og-mrk. How can I find out if there is a duplicate pull request, I tried to find one, but I wasn't able to.

**Comment by og-mrk** [comment]
> > Hello @OnyxAbove Thanks for opening this PR, but I do want to inform you that there's a PR which does this change, PR #2169 PR by @Owen-3456
> > With that being said.. if for whatever reason that PR by Owen is rejected, then this PR would probably get merged (personally, I can't see why _this_ PR might get rejected), so it's better to keep it open for Chris to review 👍
> 
> Hello, sorry for that @og-mrk. How can I find out if there is a duplicate pull request, I tried to find one, but I wasn't able to.

you can do so by:
1. Going to Pull Requests tab, and type in keywords like `ditto` in the search bar, _note you can use more advanced filter options when searching, [link to GitHub Docs](https://docs.github.com/en/issues/tracking-your-work-with-issues/filtering-and-searching-issues-and-pull-requests#using-search-to-filter-issues-and-pull-requests)_.
2. Browse through open/merged PRs in Pull Requests tab (maybe someone has done a similar change to what you intent.. but didn't put clear/concise keywords in both Title & Description of their PR).

Hope my answer is sufficient enough, and if there's anything un-clear, or might have another question, then please go ahead and ask 😄

**Comment by OnyxAbove** [comment]
@og-mrk Thank You!

**Pull request merged**